ee/tabby-db/src/users.rs

```diff
@@ -79,6 +79,11 @@ impl DbConn {
         let res = res.unique_error("User already exists")?;
         transaction.commit().await?;
 
+        self.cache.active_user_count.update(|x| *x += 1).await;
+        if is_admin {
+            self.cache.active_admin_count.update(|x| *x += 1).await;
+        }
+
```

Comment:
instead of update count, I'll prefer to simply invalidate it so we have single place to compute the count (no need to handle error, rollback etc for cache).